.github/workflows/code-coverage-comment.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     if: ${{ github.event.workflow_run.event == 'pull_request_target' && github.event.workflow_run.conclusion == 'success' }}
     steps:
       - name: Download Code Coverage
-        uses: dawidd6/action-download-artifact@v2
+        uses: actions/action-download-artifact@v4
```

Comment:
p sure this doesn't exist
https://github.com/actions/action-download-artifact

Reply:
it looks like the official action added the 'other workflows' feature
https://github.com/actions/download-artifact?tab=readme-ov-file#download-artifacts-from-other-workflow-runs-or-repositories

guessing it's a drop-in change if you switch run_id -> run-id